NEWS.md

```diff
@@ -2,6 +2,10 @@
 
 # data.table [v1.15.99](https://github.com/Rdatatable/data.table/milestone/30)  (in development)
 
+## BUG FIXES
+
+X. `dcast(fill=NULL)` only computes default fill value if necessary, which eliminates some previous warnings (for example, when fun.aggregate=min or max, warning was NAs introduced by coercion to integer range) which were potentially confusing, [#5512](https://github.com/Rdatatable/data.table/issues/5512). Thanks to Toby Dylan Hocking for the PR.
```

Comment:
move this NEWS item and cite both closed bugs

Reply:
ok